app/src/main/java/com/microsoft/device/samples/dualscreenexperience/presentation/catalog/ui/view/FirstPageContent.kt

```diff
@@ -0,0 +1,249 @@
+/*
+ *
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License.
+ *
+ */
+
+package com.microsoft.device.samples.dualscreenexperience.presentation.catalog.ui.view
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.layoutId
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.sp
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.ConstraintSet
+import com.microsoft.device.samples.dualscreenexperience.R
+import com.microsoft.device.samples.dualscreenexperience.domain.catalog.model.CatalogItem
+import com.microsoft.device.samples.dualscreenexperience.domain.catalog.model.CatalogPage
+import com.microsoft.device.samples.dualscreenexperience.presentation.catalog.utils.BottomPageNumber
+import com.microsoft.device.samples.dualscreenexperience.presentation.catalog.utils.contentDescription
+import com.microsoft.device.samples.dualscreenexperience.presentation.catalog.utils.fontDimensionResource
+import com.microsoft.device.samples.dualscreenexperience.presentation.util.sizeOrZero
+
+const val TABLE_OF_CONTENTS_ID = "tableOfContents"
+const val FIRST_PAGE_BOTTOM_PAGE_NUMBER_ID = "firstPageBottomPageNumber"
+const val FIRST_PAGE_FIRST_TEXT_ID = "firstText"
+const val FIRST_PAGE_SECOND_TEXT_ID = "secondText"
+const val FIRST_PAGE_THIRD_TEXT_ID = "thirdText"
+const val FIRST_PAGE_FOURTH_TEXT_ID = "fourthText"
+const val FIRST_PAGE_FIFTH_TEXT_ID = "fifthText"
+
+@Composable
+fun CatalogFirstPage(modifier: Modifier = Modifier, catalogList: List<CatalogItem>) {
+    val catalogItem = catalogList[CatalogPage.Page1.ordinal]
+
+    val constraintSet = getMainConstraintSet()
+
+    ConstraintLayout(constraintSet = constraintSet, modifier = modifier) {
+        TableOfContents(
+            modifier = Modifier
+                .fillMaxHeight()
+                .padding(bottom = dimensionResource(id = R.dimen.catalog_margin_normal))
+                .layoutId(TABLE_OF_CONTENTS_ID),
+            catalogItem = catalogItem
+        )
+
+        BottomPageNumber(
+            modifier = Modifier
+                .padding(
```

Comment:
Can we move the padding inside `BottomPageNumber`, so we can share between the pages?

Reply:
Yes, that makes more sense.